proc_IR_noecho.py

```diff
@@ -1,103 +0,0 @@
-from pyspecdata import *
```

Comment:
I have modified proc_IR to include arguments that make this script obsolete./ proc_IR.py does what this script does but better/more up to date